pkgs/os-specific/linux/nftables/default.nix

```
@@ -27,15 +28,16 @@ stdenv.mkDerivation rec {
   buildInputs = [
     libmnl libnftnl libpcap
     gmp jansson libedit
```

```suggestion
    gmp jansson
```
otherwise we'll still always have libedit, no?

---

Oh yeah, thanks! I've had this patch around for almost two years now, and am just now getting around to upstreaming it, so that must have got messed up in a rebase at some point.